src/database/migrations/20190909094109-create-password-resets.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
```

Comment:
The reason i used this table is when i researched on how to reset, they kept on showing that for consistency such records need to be stored, in case robots are the one that are trying to hack your account it can be seen that a lot of request are being made, so I decided also to add this too for consistency and professionalism
But you can advice me on what do better

Reply:
Good. Now I understand